config/routes.rb

```diff
@@ -3,6 +3,7 @@
   resource :session
   resource :password_reset
   resource :password
+  resource :settings
```

Comment:
I'll limit to just the edit and update actions:

`resources : settings, only: %i[edit update]`